enterprise/models.py

```diff
@@ -293,6 +293,11 @@ class Meta:
         help_text=_("Specifies whether the enterprise learner portal site should be made known to the learner.")
     )
 
+    hide_labor_market_data = models.BooleanField(
```

Comment:
I think It would be better if we keep the flag as positive instead of negative, (instead of `hide` start it with `show`)
Here are two reasons,
- It will be consistent with other existing flags, you can say they start with enable instead of disable.
- Negative flags can be confusing sometimes, 

Few suggested names are:
- enable_learner_portal_labor_market_data
- enable_labor_market_data
- show_learner_portal_labor_market_data
- show_labor_market_data

Reply:
I think the current approach is more straightforward, also it wouldn't go with business requirements as suggested in the ticket. We are already fetching a similar flag from enterprise configuration on learner portal which follows the same standard, as you can see below: 
<img width="539" alt="Screenshot 2021-10-08 at 2 32 47 PM" src="https://user-images.githubusercontent.com/55431213/136533484-5c0674fa-d84c-46e8-81db-c6c0e2ec7d17.png">